content/api/vision/coordinates.md

```diff
@@ -56,18 +56,21 @@ the camera as well as the angles which describe the spherical coordinate space.
 
 ## Orientation Coordinates
 
-Orientation represents the rotation of a marker around the x, y, and z axes. These can be accessed as follows:
+Orientation represents the rotation of a marker around the x, y, and z axes.
+These can be accessed as follows:
 
 - `rot_x_radians` / `rot_x_degrees` - the angle of rotation in radians/degrees
    counter-clockwise about the Cartesian x axis.
 - `rot_y_radians` / `rot_y_degrees` - the angle of rotation in radians/degrees
    counter-clockwise about the Cartesian y axis.
 - `rot_z_radians` / `rot_z_degrees` - the angle of rotation in radians/degrees
-   counter-clockwise about the Cartesian z axis. (tip: the z axis typically faces the camera, and thus will seem like a clockwise rotation)
+   counter-clockwise about the Cartesian z axis. (tip: the z axis typically
```

Comment:
Using the term typically suggests that there is a case where the z axis may not face the camera?
Is it worth mentioning this in the docs?

Reply:
fixed in c04d06df932b25d3425492c5e7219dd80414b5f0